src/sampler.jl

```diff
@@ -109,14 +163,13 @@ function tempered(
     kwargs...
 )
     !(adapt && typeof(swap_strategy) <: Union{RandomSwap, SingleRandomSwap}) || error("Adaptation of the inverse temperature ladder is not currently supported under the chosen swap strategy.")
-    swap_every ≥ 1 || error("`swap_every` must take a positive integer value greater ≥1.")
+    steps_per_swap ≥ 1 || error("`swap_every` must take a positive integer value greater ≥1.")
```

Comment:
Is it not clearer here to say 
```suggestion
    steps_per_swap > 0 || error("`steps_per_swap` must take a positive integer value.")
```